requirements.txt

```diff
@@ -1 +1,3 @@
 OctoPrint
+# six is already required by OctoPrint, but still add it here just in case
+six
```

Comment:
In theory, this shouldn't be needed since OctoPrint already depends on it.

But it also doesn't hurt , especially since we don't pin it to a specific version so there is no chance of a version conflict.